cmd/lakectl/cmd/commit.go

```diff
@@ -77,8 +88,9 @@ func getKV(cmd *cobra.Command, name string) (map[string]string, error) {
 func init() {
 	rootCmd.AddCommand(commitCmd)
 
-	commitCmd.Flags().StringP("message", "m", "", "commit message")
-	_ = commitCmd.MarkFlagRequired("message")
+	commitCmd.Flags().BoolVar(&allowEmptyCommitMessageFlag, AllowEmptyMessageFlagName, false, "allow an empty commit message")
+	commitCmd.Flags().StringP(MessageFlagName, "m", "", "commit message")
```

Comment:
What is the justification for marking this flag as required? In Git, omitting this flag starts an "interactive" commit, which open your text editor for a message. This would force you to add something like `-m ''` to create an empty commit message. In our case, maybe it's not necessary: maybe using the new `allow-empty-message` flag is enough.

Reply:
Not a new flag but removed it anyway.
Handling the requirements in lines 41-46